plugins/secretdata.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: secretdata
+spec:
+  version: v1.0.1
+  homepage: https://github.com/kei6u/kubectl-secretdata
+  shortDescription: A kubectl plugin for finding decoded secret data with productive search flags
+  description: |
+    This is a kubectl plugin for finding decoded secret data.
+    Since kubectl outputs base64-encoded secrets basically,
+    it makes it difficult to check the secret value.
+    And searching secrets also is difficult. This tool helps
+    verify the real secret value and find the secrets
+    you want with productive search flags.
```

Comment:
you can use up to 80 chars wide (excluding the preceding spaces), this seems quite narrow.